kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/Overview.tsx

```diff
@@ -72,36 +72,44 @@ const Overview: React.FC<Props> = ({
             </tr>
           </thead>
           <tbody>
-            {partitions?.map(({ partition, leader, offsetMin, offsetMax }) => (
-              <tr key={`partition-list-item-key-${partition}`}>
-                <td>{partition}</td>
-                <td>{leader}</td>
-                <td>{offsetMin}</td>
-                <td>{offsetMax}</td>
-                <td className="has-text-right">
-                  {!internal && !isReadOnly ? (
-                    <Dropdown
-                      label={
-                        <span className="icon">
-                          <i className="fas fa-cog" />
-                        </span>
-                      }
-                      right
-                    >
-                      <DropdownItem
-                        onClick={() =>
-                          clearTopicMessages(clusterName, topicName, [
-                            partition,
-                          ])
+            {partitions?.length !== 0 && partitions?.length !== undefined ? (
```

Comment:
```suggestion
            {partitions?.length > 0 ? (
```

Reply:
added partitions && partitions?.length > 0 becouse partitions can be undefined

Reply:
`partitions?.length > 0` should help